src/js/models/v2/awards/BaseFinancialAssistance.js

```diff
@@ -0,0 +1,117 @@
+/**
+ * BaseFinancialAssistance.js
+ * Created by Lizzie Salita 3/5/18
+ */
+
+import { formatMoney } from 'helpers/moneyFormatter';
+import CoreLocation from 'models/v2/CoreLocation';
+import BaseAwardRecipient from './BaseAwardRecipient';
+import CoreAwardAgency from './CoreAwardAgency';
+import CoreAward from './CoreAward';
+
+const BaseFinancialAssistance = Object.create(CoreAward);
+
+BaseFinancialAssistance.populate = function populate(data) {
+    // reformat some fields that are required by the CoreAward
+    const coreData = {
+        id: data.fain || data.uri,
+        internalId: data.id,
+        category: data.category,
+        startDate: data.period_of_performance_start_date,
+        endDate: data.period_of_performance_current_end_date,
+        subawardTotal: data.total_subaward_amount,
+        subawardCount: data.subaward_count
+    };
+    this.populateCore(coreData);
+
+    const recipient = Object.create(BaseAwardRecipient);
+    recipient.populate(data.recipient);
+    this.recipient = recipient;
+
+    const placeOfPerformanceData = {
+        city: data.place_of_performance.city_name,
+        county: data.place_of_performance.county_name,
+        stateCode: data.place_of_performance.state_code,
+        state: data.place_of_performance.state_name || data.place_of_performance.state_code,
+        province: data.place_of_performance.foreign_province,
+        zip5: data.place_of_performance.zip5,
+        zip4: data.place_of_performance.zip4,
+        congressionalDistrict: data.place_of_performance.congressional_code,
+        country: data.place_of_performance.country_name,
+        countryCode: data.place_of_performance.location_country_code
+    };
+    const placeOfPerformance = Object.create(CoreLocation);
+    placeOfPerformance.populateCore(placeOfPerformanceData);
+    this.placeOfPerformance = placeOfPerformance;
+
+    if (data.awarding_agency) {
+        const awardingAgencyData = {
+            name: data.awarding_agency.toptier_agency && data.awarding_agency.toptier_agency.name,
+            subtierName: data.awarding_agency.subtier_agency && data.awarding_agency.subtier_agency.name,
+            officeName: data.latest_transaction && data.latest_transaction.assistance_data
```

Comment:
Should office name be `awarding_office_name` instead of code? (etc for funding)

Reply:
Changed to `awarding_office_name` and `funding_office_name`